main.py

```diff
@@ -0,0 +1,89 @@
+import json
+import os
+import requests
+import re
+
+
+owner, repo_name = os.environ["OWNER_AND_REPO"].split("/")
+
+url = "https://api.github.com/graphql"
```

Comment:
If those string are bound not to change, move them to different file, with static content, ex: helper.py